### PR TITLE
Change the email partials to use shared parts

### PR DESCRIPTION
**Pull request by rememberlenny**
That is all

### CODE REVIEW

**Comment by jessieay** [comment]
merge conflicts on this one

**Comment by rememberlenny** [comment]
@jessieay Caught things up!

File: app/views/cancelation_mailer/cancelation_confirmation.html.haml [hunk 1]
```diff
@@ -1,42 +1,35 @@
-- header_icon = "emails/icon-pencil-circle.png"
-- top_head = t("mailer.cancelation_mailer.cancelation_confirmation.header")
+- content_for :header_icon, "emails/icon-pencil-circle.png"
+- top_head = t("mailer.cancellation_mailer.cancellation_confirmation.header")
```
Comment (jessieay) [inline app/views/cancelation_mailer/cancelation_confirmation.html.haml]: this key needs to be reverted - we are spelling it with one "l" now! :)

File: app/views/cancelation_mailer/cancelation_notification.html.haml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-- header_icon = "emails/icon-pencil-circle.png"
-- top_head = t("mailer.cancelation_mailer.cancelation_notification.header",
+- content_for :header_icon, "emails/icon-pencil-circle.png"
+- top_head = t("mailer.cancellation_mailer.cancellation_notification.header",
```
Comment (jessieay) [inline app/views/cancelation_mailer/cancelation_notification.html.haml]: same comment as above - `cancelation` please

File: app/views/cancelation_mailer/cancelation_notification.html.haml [hunk 3]
```diff
@@ -7,33 +7,29 @@
...
-            locals: { subheader:"" }
-          = render partial: "mail_shared/call_to_action/button",
-            locals: { link_path: proposal_url(@proposal), link_text: t("mailer.cancelation_mailer.cta") }
-          = render partial: "mail_shared/footer_foundation"
+  = render partial: "mail_shared/call_to_action/subheader", 
+    locals: { subheader: ''}
+  
+  = render partial: "mail_shared/call_to_action/button", 
+    locals: { link_path: proposal_url(@proposal), 
+              link_text: t("mailer.cancellation_mailer.cta")}
```
Comment (jessieay) [inline app/views/cancelation_mailer/cancelation_notification.html.haml]: `cancelation`

**Comment by jessieay** [comment]
Should we merge this after https://github.com/18F/C2/pull/1023 ? Looks like it includes the commits from that PR as well...

**Comment by rememberlenny** [comment]
@jessieay Fixed up

**Comment by jessieay** [comment]
@rememberlenny this is failing CI

File: app/views/cancelation_mailer/cancelation_confirmation.html.haml [hunk 4]
```diff
@@ -1,46 +1,35 @@
-- header_icon = "emails/icon-pencil-circle.png"
-- top_head = t("mailer.cancelation_mailer.cancelation_confirmation.header")
+- content_for :header_icon, "emails/icon-pencil-circle.png"
+- top_head = t("mailer.cancelation_mailer.cancellation_confirmation.header")
```
Comment (jessieay) [inline app/views/cancelation_mailer/cancelation_confirmation.html.haml]: this spelling is still off

File: app/views/layouts/email.html.haml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+!!!
+%html
+  %head
+    %title C2
+    %meta{:content => "text/html; charset=US-ASCII", "http-equiv" => "Content-Type"}/
+    %meta{:content => "width=device-width", :name => "viewport"}/
```
Comment (jessieay) [inline app/views/layouts/email.html.haml]: new hash syntax!

File: app/views/layouts/email.html.haml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    %link{:href => stylesheet_url("mailer_new.css"), :rel => "stylesheet", :type => "text/css"}/
```
Comment (jessieay) [inline app/views/layouts/email.html.haml]: do we still want to require this stylesheet? or can we remove this line and the stylesheet?

File: app/views/layouts/email.html.haml (hunk 5 above)
Comment (rememberlenny) [inline app/views/layouts/email.html.haml]: bleh- When I run a html->haml converter, it defaults to old. Fixing

File: app/views/layouts/email.html.haml (hunk 6 above)
Comment (rememberlenny) [inline app/views/layouts/email.html.haml]: Im going to remove!

File: app/views/cancelation_mailer/cancelation_notification.html.haml [hunk 7]
```diff
@@ -1,5 +1,5 @@
-- header_icon = "emails/icon-pencil-circle.png"
-- top_head = t("mailer.cancelation_mailer.cancelation_notification.header",
+- content_for :header_icon, "emails/icon-pencil-circle.png"
+- top_head = t("mailer.cancelation_mailer.cancellation_notification.header",
```
Comment (jessieay) [inline app/views/cancelation_mailer/cancelation_notification.html.haml]: this translation is still misspelled :(

**Pull request merged**